src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,21 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val englishGreeting = object: HelloWorldGreeting {
+            override fun greet() = "Hello world"
```

Comment:
It is a good practice to move string values to const vals or use string resources.
Please note this for the future tasks